### ISSUE TITLE
eclipseMavenCentral ecf groupId is wrong

### ISSUE THREAD

**Issue by jmini** [diffplug/goomph#162]
For all the bundles starting with `org.eclipse.ecf` the groupId is defined to be `org.eclipse.ecf` and not `org.eclipse.platform`.

See the definition in the publication file:

```xml
<mavenMappings namePattern="(org.eclipse.ecf)(.*)" groupId="$1" artifactId="$1$2"/>
```
Source: https://git.eclipse.org/c/platform/eclipse.platform.releng.git/tree/publish-to-maven-central/SDK4Mvn.aggr#n44

The eclipseMavenCentral plugin should be adjusted to handle this.

**Comment by jmini** [comment]
Reproducer:

Use this `build.gradle` file:

```gradle
buildscript {
    repositories {
        mavenCentral()
    }
    dependencies {
        classpath 'com.diffplug.gradle:goomph:3.32.1'
    }
}

apply plugin: 'com.diffplug.eclipse.mavencentral'

eclipseMavenCentral {
    release '4.14.0', {
        implementation 'org.eclipse.ecf'

        // specify this to add the native jars for this platform
        useNativesForRunningPlatform()

        // specify that all transitive dependencies should be from this release
        constrainTransitivesToThisRelease()
    }
}
```

And run `./gradlew dependencies`

You get:
```
compileClasspath - Compile classpath for source set 'main'.
\--- org.eclipse.platform:org.eclipse.ecf:3.9.4 FAILED
```

---

The correct artifact is:
```xml
<dependency>
    <groupId>org.eclipse.ecf</groupId>
    <artifactId>org.eclipse.ecf</artifactId>
    <version>3.9.4</version>
</dependency>
```

**Comment by nedtwigg** [comment]
Fixed in `3.33.1`
